.github/workflows/test-fuzz.yml

```diff
@@ -0,0 +1,78 @@
+name: Test Fuzz on Comment
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+
+env:
+  IMG_REGISTRY: ghcr.io
+  IMG_ORG: kubeflow
+  IMG_REPO: model-registry/server
+  IMG_VERSION: cicd
+  PUSH_IMAGE: false
+
+jobs:
+  check-comment:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/test-fuzz')
+    runs-on: ubuntu-latest
+    outputs:
+      should-run: ${{ steps.check.outputs.result }}
+    steps:
+      - name: Check if comment should trigger test-fuzz
+        id: check
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const comment = context.payload.comment.body.trim();
+            const isExactMatch = comment === '/test-fuzz';
+            const isLineMatch = comment.split('\n').some(line => line.trim() === '/test-fuzz');
+            return isExactMatch || isLineMatch;
+
+  test-fuzz:
+    needs: check-comment
+    if: needs.check-comment.outputs.should-run == 'true'
```

Comment:
I think you do not need `check-comment` you can simplify this, removing it completely and use the if  
```
if:  github.event.issue.pull_request && contains(github.event.comment.body, '/test-fuzz')
```

Reply:
I've updated this if check.